crates/router/src/db/tokenization.rs

```diff
@@ -0,0 +1,137 @@
+#[cfg(all(feature = "v2", feature = "tokenization_v2"))]
```

Comment:
move this to storage impl instead of keeping this in router, for can refer other table in storage_impl crate for reference